{"variants":[{"paths":["\/documentation\/passage\/oauth2api\/modelerror_oauth2callback\/invalidrequesturi"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"hierarchy":{"paths":[["doc:\/\/Passage\/documentation\/Passage","doc:\/\/Passage\/documentation\/Passage\/OAuth2API","doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback"]]},"identifier":{"url":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback\/invalidRequestUri","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"invalidRequestUri"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"}],"schemaVersion":{"major":0,"minor":3,"patch":0},"kind":"symbol","metadata":{"title":"OAuth2API.ModelError_oauth2Callback.invalidRequestUri","role":"symbol","roleHeading":"Case","modules":[{"name":"Passage"}],"externalID":"s:7Passage9OAuth2APIC25ModelError_oauth2CallbackO17invalidRequestUriyA2EmF","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidRequestUri","kind":"identifier"}],"symbolKind":"case"},"references":{"doc://Passage/documentation/Passage":{"identifier":"doc:\/\/Passage\/documentation\/Passage","type":"topic","role":"collection","title":"Passage","kind":"symbol","url":"\/documentation\/passage","abstract":[{"text":"Use Passage to add exclusively Passwordless authentication to your iOS Application.","type":"text"}]},"doc://Passage/documentation/Passage/OAuth2API":{"fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"text":"OAuth2API","kind":"identifier"}],"abstract":[],"role":"symbol","type":"topic","title":"OAuth2API","identifier":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"OAuth2API"}],"url":"\/documentation\/passage\/oauth2api"},"doc://Passage/documentation/Passage/OAuth2API/ModelError_oauth2Callback/invalidRequestUri":{"abstract":[],"title":"OAuth2API.ModelError_oauth2Callback.invalidRequestUri","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidRequestUri","kind":"identifier"}],"type":"topic","url":"\/documentation\/passage\/oauth2api\/modelerror_oauth2callback\/invalidrequesturi","kind":"symbol","identifier":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback\/invalidRequestUri"},"doc://Passage/documentation/Passage/OAuth2API/ModelError_oauth2Callback":{"fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"ModelError_oauth2Callback"}],"abstract":[{"text":"enum for parameter error","type":"text"}],"role":"symbol","type":"topic","kind":"symbol","title":"OAuth2API.ModelError_oauth2Callback","identifier":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback","navigatorTitle":[{"text":"ModelError_oauth2Callback","kind":"identifier"}],"url":"\/documentation\/passage\/oauth2api\/modelerror_oauth2callback"}}}